### PR TITLE
Add Applicant type

### PR DESCRIPTION
**Pull request by macareonie**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddApplicantCommand.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds an applicant to the talent tracker.
+ */
+public class AddApplicantCommand extends AddCommand {
+
+    public static final String COMMAND_WORD = AddCommand.COMMAND_WORD + "_applicant";
```
Comment (headcube1) [inline src/main/java/seedu/address/logic/commands/AddApplicantCommand.java:16]: Looks good! Nice way of creating a new class that inherits from the AddCommand.

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 2]
```diff
@@ -6,21 +6,16 @@
...
 import seedu.address.model.person.Person;
 
 /**
- * Adds a person to the address book.
+ * Adds a person to the talent tracker.
  */
-public class AddCommand extends Command {
+public abstract class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
```
Comment (headcube1) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:17]: Changing to abstract class. Will take note!

File: src/main/java/seedu/address/model/person/Applicant.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents an Applicant in the talent tracker.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Applicant extends Person {
+
```
Comment (headcube1) [inline src/main/java/seedu/address/model/person/Applicant.java:12]: Nice use of inheritance to create a subclass of Persons.

**Review by headcube1 — APPROVED** [review]
LGTM!

**Review by gingerbreaf — APPROVED** [review]
LGTM, good OOP design.

**Pull request merged**